cmd/convert.go

```diff
@@ -39,7 +40,9 @@ func newConvertCmd() *cobra.Command {
 		Short: "Convert between yaml and Aerospike config format.",
 		Long: `Convert is used to convert between yaml and aerospike configuration
 				files. Input files are converted to their opposite format, yaml -> conf, conf -> yaml.
-				Specifying the server version that will use the aerospike.conf is required.
+				Specifying the Aerospike Database version that will use the aerospike.conf is required unless
+				version metadata is present in the file or the --force option is used.
+				Aerospike Database metadata is written to files generated by asconfig.
```

Comment:
You should ask @carlyorke99 how he likes to refer to the server. I don't think they like to refer to it as Aerospike Database but Aerospike Server. He was trying to unify this when he first started

Reply:
I changed to using `Aerospike Database` in public facing text (help and docs) because of https://github.com/citrusleaf/tech-docs/pull/1335#discussion_r1384225564 . Aerospike Database is the official marketing term and it seems less vague than Aerospike Server to me.

Reply:
I spoke to Carl about it, he says Aerospike Database works.